.github/workflows/ci.yml

```
@@ -11,9 +11,9 @@ jobs:
         uses: actions/checkout@v2
       - name: Compile and test
         id: ci
-        uses: ignition-tooling/action-ignition-ci@bionic
+        uses: ignition-tooling/action-ignition-ci@codecov_token_refactor
```

We can move back to `bionic`

```suggestion
        uses: ignition-tooling/action-ignition-ci@bionic
```

---

ouch thanks!